Author: Ole Streicher <olebole@debian.org>
Description: Remove unneeded include to fix FTBFS on non-linux archs
--- a/casacore/casa/OS/Directory.cc
+++ b/casacore/casa/OS/Directory.cc
@@ -490,7 +490,6 @@
 
 #ifndef __APPLE__
 #include <sys/vfs.h>
-#include <linux//nfs_fs.h>
 #else
 #include <sys/param.h>
 #include <sys/mount.h>
